js/.npmrc

```diff
@@ -1,2 +0,0 @@
-save-prefix=
-engine-strict=true
```

Comment:
IIRC yarn respects `.npmrc` files too, and if so should we keep this?

Reply:
Why is it good not to have a save prefix? It means any consumer may not be able to find a version that overlaps with what they already have.

Reply:
But we can fix that later. I'll add it back for now.

Reply:
Mostly as a guard to catch when we're installing (dev) dependencies. We've promoted most of our prod dependencies to `^` explicitly.

Reply:
Dev dependencies are already guarded by the yarn lockfile. But I guess it makes sense to have specific versions so that an npm user doesn't run into problems?

Reply:
Sure but this flag controls behavior if someone does `npm i -D new-dep` (or `yarn add --dev new-dep`).

We don't want npm/yarn writing the default `new-dep@^x.y.z` to the package version in `"devDependencies"`.